core/src/main/java/org/frankframework/logging/IbisNdcPatternConverter.java

```diff
@@ -28,7 +28,7 @@
 public class IbisNdcPatternConverter extends LogEventPatternConverter {
 
 	private IbisNdcPatternConverter() {
-		super("IbisNdcPatternConverter", "ndc");
+		super("IbisNdcPatternConverter", "TC");
```

Comment:
I don't know what the difference between 'ndc' and 'TC' is. Can you perhaps add a comment on this line on TC?

Reply:
A self made up shorthand for `PatternConverter`.